bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/handler/SoulissT13Handler.java

```diff
@@ -65,12 +60,6 @@ public void setState(@Nullable PrimitiveType state) {
     public void handleCommand(ChannelUID channelUID, Command command) {
         if (command instanceof RefreshType) {
             switch (channelUID.getId()) {
-                case SoulissBindingConstants.STATEONOFF_CHANNEL:
```

Comment:
I believe constant `SoulissBindingConstants.STATEONOFF_CHANNEL` can also be removed.